src/server/kiwix-serve.cpp

```diff
@@ -49,8 +49,8 @@ static const char USAGE[] =
 R"(Deliver ZIM file(s) articles via HTTP
 
 Usage:
- kiwix-serve [options] ZIMPATH ...
- kiwix-serve [options] (-l | --library) LIBRARYPATH
+ kiwix-serve [options]... [ZIMPATH]...
+ kiwix-serve [options]... (-l | --library) LIBRARYPATH
```

Comment:
This is not a string that is just output to the user as is. It is also used by `docopt` to derive the syntax of the command line usage of `kiwix-serve` and parse the options and command line arguments. There are no automated tests invoking `kiwix-serve` with various options and arguments. Have you tested that this seemingly minor change doesn't confuse `docopt`?